test/integration/services/Common.go

```diff
@@ -0,0 +1,42 @@
+package services
+
+import (
+	"flag"
+	"fmt"
+	"github.com/onsi/ginkgo/config"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"os"
+)
```

Comment:
It's good practice to separate import paths in three different groups: Standard library imports / internal packages imports / external packages imports
https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Yep, @A-Hilaly is correct (and I've been following this pattern in the rest of the codebase). Depending on your editor or IDE of choice, you can typically find a module or plugin that autoformats this kind of thing for you. For instance, I use vim, and the vim-go support automatically runs `go fmt` whenever I save a Go file. Makes things like this a non-issue ;)

Reply:
oh ok. i just allowed GoLand to take care of it. but sure will fix it .ty